examples/airflow_orchestration/README.md

```diff
@@ -68,16 +71,15 @@ This will bootstrap Airflow, start up all the necessary components and run them
 in the background. When the setup is finished, it will print username and
 password for the Airflow webserver to the console.
 
-{% hint style="warning" %} If you can't find the password on the console, you
+WARNING: If you can't find the password on the console, you
```

Comment:
Is there a reason why the hint style was removed?

Reply:
Yes. It doesn't render anywhere. Just shows up as plain text garbage.